src/scraper.py

```diff
@@ -122,7 +147,7 @@ def setup_url(searchurl: str, imgsize: str, imgcolor: str, imgtype: str, safesea
         features += ["tbs=" + delim2.join(subfeatures[0])]
     if (subfeatures[1] != []):
         features += ["safe=" + delim2.join(subfeatures[1])]
-    
+    print(delim1.join(features))
```

Comment:
Please wrap debugging print statements in an 'if DEBUG' and provide them useful text to identify the output